src/components/common/NameRequestInfo.vue

```diff
@@ -167,38 +166,30 @@ export default class NameRequestInfo extends Mixins(DateMixin, EnumMixin) {
   readonly NOT_REQUIRED_STATE = 'Not Required'
   readonly WAIVED_STATE = 'Waived'
 
-  // Global Store
-  @State(state => state.stateModel.nameTranslations)
-  readonly nameTranslations!: NameTranslationIF[]
-
-  // Global Actions
   @Action setNameTranslationState!: ActionBindingIF
 
-  // Global getters
-  @Getter isEntityType!: GetterIF;
   @Getter getEntityType!: CorpTypeCd
-  @Getter getTempId!: GetterIF;
-  @Getter getNameRequestNumber!: GetterIF;
-  @Getter getNameRequestDetails!: NameRequestDetailsIF;
-  @Getter getNameRequestApplicant!: NameRequestApplicantIF;
-  @Getter getNameTranslations!: NameTranslationIF[];
+  @Getter getNameRequestNumber!: string
+  @Getter getNameRequestDetails!: NameRequestDetailsIF
+  @Getter getNameRequestApplicant!: NameRequestApplicantIF
+  @Getter getNameTranslations!: NameTranslationIF[]
 
-  /** The entity title  */
+  /** The entity title.  */
   private get getEntityTypeDescription (): string {
     return `${this.getCorpTypeDescription(this.getEntityType)}`
   }
 
-  /** The request type */
-  private requestType (): string {
+  /** The request type. */
+  private get requestType (): string {
```

Comment:
What's the difference here?

Reply:
What it returns is static so I changed it to a getter, which can have some performance improvements (eg, caching, and reactive).
The main difference is how you use it. It used to be `requestType()` (ie, method) and now it's `requestType` (ie, property).